examples/basic-embed/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Data Layer Observer Example: basic embed</title>
+    <meta charset="utf-8" />
+    <script>
+      /* These are examples of static data that will be read on load */
+      window.exampleData = {
+        page: {
+          pageName: 'basic embed',
+          pageInfo: {
+            pageID: '1234'
+          }
+        }
+      }
+      window.adInformation = {
+        accountID: 'abcd'
+      }
+      window.ops = {
+        clusterInfo: {
+          id: '444-111-11231',
+          load: 0.4
+        }
+      }
+
+      /* This function will receive information as the DLO rules find it */
+      function ruleDestination(info) {
+        console.log('ruleDestination received:', info);
+      }
+    </script>
+  </head>
+  <body>
+    <h1>Data Layer Observer Example: basic embed</h1>
+
+    <p>Take a look at the source code of this page to see how the DLO reads rules and configuration.</p>
+    <p>The Javascript console will show you output that demonstrates the rules in action.</p>
+
+    <script>
+      // OperatorOptions that is always used just before before the destination
+      // Default is null
+      window['_dlo_beforeDestination'] = null;
+
+      // Redirects output from a destination to previewDestination when testing rules
+      // Default is false
+      window['_dlo_previewMode'] = true;
+
+      // The output destination using rule selection syntax for use with previewMode
+      // Default is null
+      window['_dlo_previewDestination'] = 'console.log';
+
+      // When true reads data layer target(s) and emits the initial value(s)
+      // Default is false
+      window['_dlo_readOnLoad'] = true;
+
+      // When true validates rules to prevent processing invalid options
+      // Default is false
+      window['_dlo_validateRules'] = false;
```

Comment:
I'm thinking since `previewMode` is set to true that we also set `_dlo_validateRules` set to true.  So the example is in "safe mode".